packages/graph-explorer/src/modules/NodeExpand/NodeExpandContent.tsx

```diff
@@ -151,3 +155,9 @@ function ExpandButton({
     </Button>
   );
 }
+
+function firstNeighborAvailableForExpansion(
```

Comment:
what is return type of this function?

Reply:
It is inferred to be `NeighborOption | undefined`